### DISCUSSION TITLE
Google doesn't index our pages even though they been discovered or crawled

### DISCUSSION THREAD

**Discussion by patriciomacadden** [vercel/community#4953] [Help]
### Summary

Hi! First of all, I'm sorry if this question has already been asked. I haven't seen an issue like this one but if there is please let me know.

We have an institutional website (with a blog) for our company under the hobby plan. It's been running for more than a year and it's SEO friendly (98% score in semrush). Lighthouse score is all green, and we have set up the google search console. Google has discovered and crawled all pages, but only indexed 55 (we have no more than 90 pages in total). 

More context:
* we're using next-sitemap and the sitemap has been submitted to google search console
* we're using next-seo
* we've implemented schema.org
* lighthouse: all green

I've read a lot and done everything already but still, I have two main issues:

1. it won't index 25 pages or so
2. if I do a `site:` search, it won't bring 55 results, it will bring only 20-ish

I've read in Google search console's help that Google might delay the crawl if it thinks it could overload the server.

Does Vercel apply any kind of penalty, give less priority, or do something that might affect speed to hobby plans for that to happen? Instinctively I think you don't but having tried everything it's something that I think could be possible.

If anyone has any hints I would really appreciate it!

Thanks in advance!

### Steps to Reproduce

I wouldn't know how to reproduce it :P. Hopefully no one else experiments this.

**Comment by gvheertum** [comment]
Interesting question, currently I also have quite a hard time getting sites indexed on Google, while Bing and DuckDuckGo seem to have no problem getting the sites indexed. I first thought this might be due to Google indexing being extremely picky or that the battle against content farms and bots seems to be a tough struggle, but I begin to think migrating the specific sites to NextJS and Vercel might be a (small) part of the problem. 

Problem is that parts of the blog/site are new (or even the domains are quite recent), so I'm not sure these would have been indexed if hosted anywhere else or if the platform or technique might be a part of the cause. If there is some kind of penalty on the hobby plan, that's fine as long if that is stated clearly. In that case I would consider the Pro plan, but if I'm not sure that would solve the issue I will be reluctant on doing that.

Is this still a problem for your sites? Or did it gradually resolve over time?
